airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/accounts.json

```diff
@@ -42,7 +42,7 @@
       "type": ["null", "number"]
     },
     "customer.optimization_score_weight": {
-      "type": ["null", "number"]
+      "type": ["null", "number", "integer"]
```

Comment:
I’m not sure that we support two types for one field

Reply:
But we also have this type of matching in a few different connectors: 
[source-clockify](https://github.com/airbytehq/airbyte/blob/a277025a8cc6c86ee382353c8542141a44f06405/airbyte-integrations/connectors/source-clockify/source_clockify/schemas/projects.json)
[source-amplitude](https://github.com/airbytehq/airbyte/blob/f5321d18953a4dd5a6f7448ee0bddc23929656cc/airbyte-integrations/connectors/source-amplitude/source_amplitude/schemas/events.json) 
And other
It also doesn't raise errors.

Reply:
I believe we can update typing during parse response.

Reply:
Fixed